tools/cli/src/index.ts

```diff
@@ -335,39 +413,137 @@ program
     },
   );
 
-// Remove command
-program
-  .command('remove')
-  .description('Removes a card, a template or an attachment')
+const rank = program.command('rank');
+
+rank
+  .command('card')
+  .description('Set the rank of a card to be after another card')
+  .argument('<cardKey>', 'Cardkey of the card to be moved')
   .argument(
-    '<type>',
-    `Possible types: ${Commands.removableTypes.join(', ')}`,
-    parseRemovableTypes,
+    '<afterCardKey>',
+    'Cardkey of the card that the card should be after',
+  )
+  .option('-p, --project-path [path]', `${pathGuideline}`)
+  .action(
+    async (cardKey: string, afterCardKey: string, options: CardsOptions) => {
+      const result = await commandHandler.command(
+        Cmd.rank,
+        ['card', cardKey, afterCardKey],
+        options,
+      );
+      handleResponse(result);
+    },
+  );
+
+rank
+  .command('rebalance')
+  .description(
+    'Rebalance the rank of all cards in the project. Can be also used, if ranks do not exist',
   )
   .argument(
-    '<targetName>',
-    'Resource that should be removed; template name or cardkey. If removing an attachment, define cardkey of owning card.',
+    '[parentCardKey]',
+    'if null, rebalance the whole project, otherwise rebalance only the direct children of the cardkey',
   )
-  .argument('[detail]', 'attachment name to remove')
+  .option('-p, --project-path [path]', `${pathGuideline}`)
+  .action(async (cardKey: string, options: CardsOptions) => {
+    const result = await commandHandler.command(
+      Cmd.rank,
+      ['rebalance', cardKey],
+      options,
+    );
+    handleResponse(result);
+  });
+
+// Remove command
+const remove = program.command('remove');
+
+remove
+  .command('attachment')
+  .argument('<cardKey>', 'cardKey of the owning card') // add unlimited number of details
```

Comment:
"// add unlimited number of details"
I don't understand the comment.

Reply:
I removed the unrelated comment